src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java

```diff
@@ -59,7 +67,11 @@ public String evaluate(Run<?, ?> run, FilePath workspace, TaskListener listener,
             return "No tests ran.";
         }
 
-        int failCount = testResult.getFailCount();
+        List<TestResult> failedTests = testResult.getFailedTests().stream().collect(Collectors.toList());
+
+        failedTests = testSuiteName.length() == 0 ? failedTests : failedTests.stream().filter(t -> t.getFullName().contains(testSuiteName)).collect(Collectors.toList());

```

Comment:
I think it would be better if you could use a regular expression to filter instead of just a single name. Also, would people perhaps want to filter on multiple names?

Reply:
That is a good idea. Should I add 2 parameters, one for regex and one for multiple names ?